lightning/src/util/ser_macros.rs

```diff
@@ -26,7 +26,7 @@ macro_rules! encode_tlv {
 }
 
 macro_rules! encode_tlv_stream {
-	($stream: expr, {$(($type: expr, $field: expr, $fieldty: ident)),*}) => { {
+	($stream: expr, {$(($type: expr, $field: expr, $fieldty: ident)),* $(,)*}) => { {
```

Comment:
I believe we support versions of rust prior to the introduction of `?`.

Reply:
Looks like it made it into 1.32.

https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1320-2019-01-17

Reply:
Ah, maybe it would work now and my information is out of date? In any case, we use * in a number of places and this particular commit has already landed upstream on another PR.